frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1874,6 +1984,28 @@ function ReaderHighlight:editHighlightStyle(page, i)
     self:showHighlightStyleDialog(apply_drawer, item.drawer)
 end
 
+function ReaderHighlight:editHighlightColor(page, i)
+    local item = self.view.highlight.saved[page][i]
+    local apply_color = function(color)
+        self:writePdfAnnotation("delete", page, item)
```

Comment:
Not familiar with all this, just asking: does the "find it out in the PDF" part use the style & color to find the annotation to delete? or just the boxes? If I change all styles with `Apply default style to all highlights` (which does not update PDF annotations, which is ok if we don't do that for other styles), will this still find them and allow changeing individual colors? or will these annotations be "orphans"?
To be clear: let all the mess be if it already exists for annot_type.

Reply:
Honestly I haven't tested the pdf code yet. I only use epub files on my device.
I suppose pdf annotations should also be updated as part of the "Apply default style" button. I'll add that as a mental todo for the next MR revision.

Reply:
> use the style & color to find the annotation to delete? or just the boxes?

pboxes only